pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/OffersSuggestions/OffersSuggestions.tsx

```diff
@@ -52,16 +52,16 @@ const getPossibleFilterValues = (
   const possibleFacetFilters = []
   //  The returned array should be in the order of preferred suggestion filters
 
-  if (formValues.eventAddressType !== OfferAddressType.OTHER) {
+  if (formValues.categories.length !== 0) {
     possibleFacetFilters.push({
       values: {
         ...formValues,
         eventAddressType: OfferAddressType.OTHER,
+        domains: [],
+        categories: [],
         geolocRadius: DEFAULT_GEO_RADIUS,
       },
-      headerMessage: (
-        <>Découvrez des offres qui relèvent d’autres types d’intervention</>
-      ),
+      headerMessage: <>Découvrez des offres qui relèvent d’autres catégories</>,
```

Comment:
ça marche pas avec des guillemets ? 
```suggestion
      headerMessage: "Découvrez des offres qui relèvent d’autres catégories",
```